django/mgmt/utils.py

```diff
@@ -28,6 +28,11 @@ def gen_key(ps):
     add_v  = [t,t,f]
     del_v  = [t,t,t]
 
+    # read_metadata, add_metadata, update_metadata, delete_metadata
+    read_m = [t, f, f, f]
+    add_m = [t, t, f, f]
+    del_m = [t, t, t, t]
+ 
     def make_selection(p,is_chan):
```

Comment:
I know you didn't write this, but can you document what `p` is for future readers?